koocook_core/models/user/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 
 def _default_preferences():
-    return dict()
+    return dict
```

Comment:
You'll need to change this as well, as when `default` call the function, it will return the `dict` `type` instead of an instance.